src/TheSpace/TheSpace.sol

```diff
@@ -79,10 +79,38 @@ contract TheSpace is HarbergerMarket {
 
     /**
      * @notice Get color for a pixel.
-     *
-     * @dev Emits {Color} event.
      */
     function getColor(uint256 tokenId) public view returns (uint256) {
         return pixelColor[tokenId];
     }
+
+    /**
+     * @notice Get owned tokens for a user.
+     *
+     * @dev offset based pagination
+     */
+    function getTokensByOwner(
```

Comment:
Shall we add another interface to batch query pixels?

Reply:
Do you mean adding `getPixelsByOwner` besides adding `getTokensByOwner` ？